index.js

```diff
@@ -207,35 +207,55 @@ const artists = [
 
 (1) Name of the first artist (0th index) in the array
 (2) Bio of the third artist (2nd index) in the array */
-
-
+console.log(artists[0].name);
+console.log(artists[2].bio);
 
 /* Task 2: There is a typo in your dataset 😱 The 9th artist, Vincent Van Gogh is currently Vincent Van Dough. Use an array method to fix this issue and console.log() to check your work. */
-
+function fixTypo(array,index,  name){
+  array[index].splice(8, 1)
+  array[index].push({name})
+  return array;
+}
+console.log(artists, 0 ,name = "Vincent Van Gogh");
 
 
 /* Task 3: Create a function called `getArtistByIndex` that takes two arguments:
  *     (1) artists array
  *     (2) a number which is the desired index in the array.
  * getArtistByIndex returns a string in the format `The artist at index {id} is {name}.`
- * 
- * For example, if getArtistByIndex is invoked with the artists dataset and the number 0,
+ */
+
+ /* For example, if getArtistByIndex is invoked with the artists dataset and the number 0,
  * it will return `The artist at index 0 is Amedeo Modigliani`.
 */
-function getArtistByIndex(array, index) {
-    /* code here */
-  }
-  
+function getArtistByIndex(array, index, id, name, string){
+  array[index].push({id})
+  array[index].push({name})
+  array[index].push({string}) 
+  return array[index], ({string});
+}
+console.log(artists, 0, 0,  {string}= `The artist at ${0} is ${`Amedeo Modigliani`}.`);
```

Comment:
This function should be returning the string with the name coming from the item in the array. It should not be console logged by manually typing in the name